scripts/Components/Map.jsx

```diff
@@ -23,16 +23,168 @@ function Map() {
   function statesCustomConfig() {
     return {
       NJ: {
-        fill: 'navy',
```

Comment:
just wondering, would it be easier to have the colors saved as variables and just fill it that way? in case we want to change it again.